src/pages/index.astro

```diff
@@ -86,89 +88,40 @@ import { Icon } from 'astro-icon';
+                    you won't be left in the dark.
+                </HomepageCard>
             </div>
             <div class="col">
-                <div class="card h-100 shadow-sm">
-                    <div class="card-body d-flex align-items-center">
-                        <div>
-                            <h5 class="card-title">CI Testing</h5>
-                            <p class="card-text">
-                                Every time a change is made to the pipeline code, nf-core pipelines use
-                                continuous-integration testing to ensure that nothing has broken.
-                            </p>
-                        </div>
-                        <Icon name="logos:github-actions" class="ms-3" />
-                    </div>
-                </div>
+                <HomepageCard title="CI Testing" icon="logos:github-actions">
+                    Every time a change is made to the pipeline code, nf-core pipelines use continuous-integration
+                    testing to ensure that nothing has broken.
+                </HomepageCard>
             </div>
             <div class="col">
-                <div class="card h-100 shadow-sm">
-                    <div class="card-body d-flex align-items-center">
-                        <div>
-                            <h5 class="card-title">Stable Releases</h5>
-                            <p class="card-text">
-                                nf-core pipelines use GitHub releases to tag stable versions of the code and software,
-                                making pipeline runs totally reproducible.
-                            </p>
-                        </div>
-                        <Icon name="line-md:check-list-3-twotone" class="ms-3" color="#82D948" />
-                    </div>
-                </div>
+                <HomepageCard title="Stable Releases" icon="line-md:check-list-3-twotone" icon_color="#82D948">
+                    nf-core pipelines use GitHub releases to tag stable versions of the code and software, making
+                    pipeline runs totally reproducible.
+                </HomepageCard>
             </div>
             <div class="col">
-                <div class="card h-100 shadow-sm">
-                    <div class="card-body d-flex align-items-center">
-                        <div>
-                            <h5 class="card-title">Packaged software</h5>
-                            <p class="card-text">
-                                Pipeline dependencies are automatically downloaded and handled using Docker,
-                                Singularity, Conda or others. No need for any software installations.
-                            </p>
-                        </div>
-                        <i class="fad fa-box-alt fa-5x text-info ms-3"></i>
-                    </div>
-                </div>
+                <HomepageCard title="Packaged software" fontawesome="fad fa-box-alt text-info">
```

Comment:
I was wondering if we should use this rewrite to switch to the fontawesome6 syntax: e.g.:
```suggestion
                <HomepageCard title="Packaged software" fontawesome="fa-duotone fa-box-taped text-info">
```

Reply:
Done, site-wise. Will probably need to keep doing it as we copy over more HTML..